src/API/src/db/bionomics/entities/bionomics.entity.ts

```diff
@@ -47,6 +52,10 @@ export class Bionomics extends BaseEntity {
   @Field({ nullable: true })
   insecticide_control: boolean;
 
+  @Column('boolean', { nullable: true })
+  @Field({ nullable: true })
+  ITN_use: boolean;
```

Comment:
Can this be changed to `itn_use` - I know it's an abbreviation, but we should stick to the snake casing